package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ticket-evolution-seatmaps",
-  "version": "0.5.4",
+  "version": "0.5.6",
```

Comment:
What happened to `0.5.5`?

Reply:
I think I forgot to update the package.json for the last release.